src/main/java/ca/intelliware/ihtsdo/mlds/web/rest/AtomFeedResource.java

```diff
@@ -1,5 +1,6 @@
 package ca.intelliware.ihtsdo.mlds.web.rest;
 
+
```

Comment:
@dmcgihtsdo While trying out random solutions for the SB3 Upgrade. Some imports were added and removed, because of this some empty lines are added which caused this issue. We have addressed it now.